src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -121,7 +122,7 @@ export default [
     {
         textTranslateKey: 'reportActionContextMenu.copyLink',
         icon: Expensicons.LinkCopy,
-        shouldShow: () => true,
+        shouldShow: (type, reportAction, betas) => Permissions.canUseCommentLinking(betas),
```

Comment:
We don't need `type` and `reportAction`

Reply:
We're using separate arguments instead of object as param. Hence to use the third param, we have to pass the first and the second one.